src/PointwiseFunctions/MathFunctions/RegisterDerivedWithCharm.cpp

```diff
@@ -5,8 +5,14 @@
 
 #include "PointwiseFunctions/MathFunctions/Gaussian.hpp"
 #include "PointwiseFunctions/MathFunctions/MathFunction.hpp"
+#include "PointwiseFunctions/MathFunctions/PowX.hpp"
+#include "PointwiseFunctions/MathFunctions/Sinusoid.hpp"
 #include "Utilities/Serialization/RegisterDerivedClassesWithCharm.hpp"
 
+namespace MathFunctions {
 void register_derived_with_charm() {
-    register_classes_with_charm<MathFunctions::Gaussian<1, Frame::Inertial>>();
+  register_classes_with_charm<Gaussian<1, Frame::Inertial>>();
+  register_classes_with_charm<PowX<1, Frame::Inertial>>();
+  register_classes_with_charm<Sinusoid<1, Frame::Inertial>>();
```

Comment:
The `register_classes_with_charm` function can actually take a parameter pack so just do one function call with all the templates separated by `,`.